src/electionguard-ui/ElectionGuard.UI/Controls/ContestControl.xaml.cs

```diff
@@ -5,41 +5,5 @@ public partial class ContestControl : ContentView
     public ContestControl()
     {
         InitializeComponent();
-        BindingContext = this;
-        GenerateSelections();
     }

```

Comment:
Removed all of the code that was going to generate the data for the control and switched to using a ContestItem that gets filled in when the tally page gets loaded.